lime/lime_text.py

```diff
@@ -63,7 +63,7 @@ def visualize_instance_html(self, exp, label, div_name, exp_object_name,
         if not text:
             return u''
         text = (self.indexed_string.raw_string()
-                .encode('ascii', 'xmlcharrefreplace').decode())
+                .encode('utf-8', 'xmlcharrefreplace').decode())
```

Comment:
This would break non ascii code, I think. If we try
```
text = u'This is a weird text \xe9'
text.encode('utf-8', 'xmlcharrefreplace')
```
we get
`UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 21: ordinal not in range(128)
`

Reply:
The problem occurred only in python2. Now I fixed it.

**Explaination:**
I ran your example code to reproduce the error but it worked fine. 

```
text = u'This is a weird text \xe9'
text.encode('utf-8', 'xmlcharrefreplace')
```
```
'This is a weird text \xc3\xa9'
```
However UnicodeDecodeError still occured when running lime with unicode text in python2. The problem was with the decode function, not with the encode function. I changed encoding parameter of the decode function. As long as the input is unicode string, now it works fine.

The only problem remaining is that unicode characters don’t show up on the right panel of the show_in_notebook explanation in python2. (they show up correctly in python3) I’m not sure how to fix it yet but at least now the code doesn't produce any errors and highlighted words are shown correctly.

Reply:
Yeah, I notice now that my example was missing a .decode() at the end.